tests/integration/security/testdata/authz/v1beta1-tcp.yaml.tmpl

```diff
@@ -1,86 +1,86 @@
-# The following policy denies request with path "/data" to port 8090 for workload b
+# The following policy denies request with path "/data" to port 8091 for workload
 
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-b-deny
+  name: policy-dst0-deny
   namespace: "{{ .Namespace }}"
 spec:
   selector:
     matchLabels:
-      "app": "b"
+      "app": "{{ .dst0 }}"
   action: DENY
   rules:
   - to:
     - operation:
         paths: ["/data"]
-        ports: ["8090"]
+        ports: ["8091"]
 ---
 
 # The following policy denies:
-# request to port 8090 for workload c
+# request to port 8090 for workload dst1
 # request to port 8093 with principal suffix matching
 # request to port 8092 with namespace suffix matching
 
 apiVersion: security.istio.io/v1beta1
 kind: AuthorizationPolicy
 metadata:
-  name: policy-c-deny
+  name: policy-dst1-deny
   namespace: "{{ .Namespace }}"
 spec:
   selector:
     matchLabels:
-      "app": "c"
+      "app": "{{ .dst1 }}"
   action: DENY
   rules:
   - to:
     - operation:
-        ports: ["8090"]
+        ports: ["8091"]
   - to:
     - operation:
-        ports: ["8093"]
+        ports: ["8094"]
     from:
     - source:
-        principals: ["*/ns/{{ .Namespace }}/sa/b"]
+        principals: ["*/ns/{{ .Namespace }}/sa/{{ .src1 }}"]
```

Comment:
we should stick to a single variable for the same workload, for workload `b`, it was mapped to `dst0` and should always be so.

And if you follow the above comment that just use the `a`, `b`, in the name and variable, you won't have this problem at all.

Reply:
This policy change is reverted, now the workloads are parameterized. e.g. `a`->`{{ .a }}`.